docs/building-linux-image.md

```diff
@@ -108,7 +114,46 @@ The following builds an extension image for the `elemental3-toolkit` command lin
         └── elemental3-toolkit-3.0.x86-64.raw
     ```
 
-*Prepare the `elemental3-toolkit-3.0.x86-64.raw` extension image as an overlay:*
+6. The `mkosi.output/elemental3-toolkit-3.0.x86-64.raw` file is the system extension image that can be used during the OS installation process following the steps in [Prepare the system extension image as an overlay](#prepare-the-system-extension-image-as-an-overlay).
+
+
+#### Install RPMs in a systemd-extension image
```

Comment:
Change systemd-extension -> system extension, since it seems to be the preferred term in this document.